util.go

```diff
@@ -27,10 +27,15 @@ type ConfigParseError struct {
 }
 
 // Error returns the formatted configuration error.
-func (pe ConfigParseError) Error() string {
+func (pe *ConfigParseError) Error() string {
```

Comment:
Why make it a pointer?

Reply:
Honestly I don't remember right now. I came from yaml issue referenced above and had some restriction with non-pointer values. Started discussion in https://groups.google.com/u/1/g/golang-nuts/c/3M3QDkochcs/m/uo-x6jA0BgAJ which concluded in 

> however i think it is a good default position to use pointer values to 
implement interfaces generally, if there is no other good reason not to. 

If you're not happy with pointer type I'll need to recreate the entire case and try to find out what the root cause issue with non-pointer types was.

Reply:
Honestly, I'm not happy doing it unless I understand why it's necessary.

Reply:
I have reverted the pointer type and added a test- it does unwrap as expected.